docs/guidance/map/milestones.md

```diff
@@ -0,0 +1,177 @@
+
+```eval_rst
+
+.. jsoninclude:: ../../examples/milestones/implementation-milestones-1.json
+   :jsonpointer: 
+   :expand: releases, contracts, implementation, milestones
+   :title: implementation
+
+```
+
+```eval_rst
+
+.. jsoninclude:: ../../examples/milestones/implementation-milestones-2.json
+   :jsonpointer: 
+   :expand: releases, contracts, implementation, milestones
+   :title: implementation-update-1
+
+```
+
+```eval_rst
+
+.. jsoninclude:: ../../examples/milestones/implementation-milestones-3.json
+   :jsonpointer: 
+   :expand: releases, contracts, implementation, milestones
+   :title: implementation-update-2
+
+```
+
+#### Delivery and financing data
+
+This example shows how milestones can be used to keep track of delivery and financing (payment) data in a contracting process.
+
+The example below includes three OCDS releases:
+
+* An implementation release with contract information along its implementation milestones.
+* An implementation update release with the actual date the milestone was reached.
+* An implementation update release with payment information
+
+In the implementation release:
+
+* Milestones have been set for the delivery and payment of goods and services related to the project. Contract information is released along its implementation milestones.
+
+In the first implementation update release:
+
+* The milestone ("Finish the exterior and interior walls") has been reached, then the `status` field is set to 'met' and the relevant dates added to the `dateMet` and `dateModified` fields.
+
+In the second implementation update release:
+
+* The construction company has received payment for the work done so far, and the milestone for the wall restoration with type 'financing' is updated. A new `transaction` is disclosed, with the amount paid to the company.
+
+```eval_rst
+
+.. jsoninclude:: ../../examples/milestones/af-implementation-milestones-1.json
+   :jsonpointer: 
+   :expand: releases, contracts, implementation, milestones
+   :title: implementation
+
+```
+
+```eval_rst
+
+.. jsoninclude:: ../../examples/milestones/af-implementation-milestones-2.json
+   :jsonpointer: 
+   :expand: releases, contracts, implementation, milestones
+   :title: implementation-update-1
+
+```
+
+```eval_rst
+
+.. jsoninclude:: ../../examples/milestones/af-implementation-milestones-3.json
+   :jsonpointer: 
+   :expand: releases, contracts, implementation, milestones, transactions
+   :title: implementation-update-2
+
+```
+
+Notes:
+
+* Contract implementation milestones are kept updated as things progress with the relevant date added to the `dateModified` field.
+* At the point of contract signature, a comparison between `tender/milestones` and `contract/implementation/milestones` with a `milestone/type` of 'delivery' or 'reporting' should reveal how the negotiated contract differs from what was set out at tender time.
```

Comment:
Move to the introductory section, after the sentence introducing the `.type` field.